airbyte_cdk/sources/types.py

```diff
@@ -15,9 +16,17 @@
 
 
 class Record(Mapping[str, Any]):
-    def __init__(self, data: Mapping[str, Any], associated_slice: Optional[StreamSlice]):
+    def __init__(
+        self,
+        data: Mapping[str, Any],
+        associated_slice: Optional[StreamSlice] = None,
+        stream_name: Optional[str] = None,
```

Comment:
I think we should make this non optional and just on the filtering, we set a dummy value. I would like the typing to confirm that we pass this value wherever we can. WDYT?

Reply:
hmm, i'll check it

Reply:
converted to non optional